content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -465,12 +465,13 @@ metadata:
 
 #### Internal Behavior
 This admission controller has the following behavior:
+
   1. If the `Namespace` has an annotation with a key `scheduler.alpha.kubernetes.io/node-selector`, use its value as the
      node selector.
-  1. If the namespace lacks such an annotation, use the `clusterDefaultNodeSelector` defined in the `PodNodeSelector`
+  2. If the namespace lacks such an annotation, use the `clusterDefaultNodeSelector` defined in the `PodNodeSelector`
```

Comment:
Please consider unindent line 469-474. Looks to me the problem was caused by the extra leading spaces in those lines.